spec/controllers/msa_components_controller_spec.rb

```diff
@@ -12,4 +12,18 @@
       expect(response).to have_http_status(:success)
     end
   end
+
+  describe "GET #new" do
+    it "returns http redirect with other users" do
+      stub_auth
+      get :new
+      expect(response).to have_http_status(:redirect)
```

Comment:
if this is testing the unauthorized flow, might be worth to have an expectation for the flash error message (and it would also made the test clearer)